### PR TITLE
doc(serverless): add workaround for return data that is array/list

### PR DESCRIPTION
**Pull request by appletreeisyellow**
Closes https://github.com/influxdata/idpe/issues/18775

Rationale for this document is here: https://github.com/influxdata/idpe/issues/18775#issuecomment-2297609325

The added doc looks like this:
Screenshot 2024-08-20 at 8 41 34 AM

_Describe your proposed changes here._

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-serverless/query-data/troubleshoot-and-optimize/troubleshoot.md [hunk 1]
```diff
@@ -28,6 +28,23 @@ If a query doesn't return any data, it might be due to the following:
 - Your data falls outside the time range (or other conditions) in the query--for example, the InfluxQL `SHOW TAG VALUES` command uses a default time range of 1 day.
 - The query (InfluxDB server) timed out.
 - The query client timed out.
+- The query return type is not supported by the client library. For example, array / list type may not be supported, in this case convert the result into a string using `array_to_string()`. For example,
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/query-data/troubleshoot-and-optimize/troubleshoot.md:31]:
```suggestion
- The query return type is not supported by the client library.
  For example, array or list types may not be supported.
  In this case, use `array_to_string()` to convert the array value to a string--for example:
```

File: content/influxdb/cloud-serverless/query-data/troubleshoot-and-optimize/troubleshoot.md [hunk 2]
```diff
@@ -28,6 +28,23 @@ If a query doesn't return any data, it might be due to the following:
...
+```
+
+can be rewrote into
+
+
+```sql
+SELECT array_to_string(array_agg([1, 2, 3]), ', ')
+```
+
+{{% /code-tab-content %}}
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/query-data/troubleshoot-and-optimize/troubleshoot.md:46]:
~~~suggestion
```sql
SELECT array_to_string(array_agg([1, 2, 3]), ', ')
```
~~~

**Review by sanderson — CHANGES_REQUESTED** [review]

**Review by sanderson — APPROVED** [review]
Thanks @appletreeisyellow! This is good to go!

**Comment by appletreeisyellow** [comment]
Thank you @sanderson!

**Pull request merged**